### ISSUE TITLE
Exclude source data checkbox does not work when exporting to halez in 3.4.1

### ISSUE THREAD

**Issue by Kate-Lyndegaard** [halestudio/hale#714]
I would like to export as halez without source data. I checked the box to exclude source data, however the source data is included in the export. I verified this twice with my transformation project.

**Comment by florianesser** [comment]
@Kate-Lyndegaard Does the source data happen to be a Shapefile? Because in that case the Shapefile is also the source of the schema and, therefore, won't be excluded if you check the `Exclude source data` box.

If you want to publish the project to hale connect, you can use hale studio's upload mechanism and then check both `Exclude source data` and `Use cached internal schema representation`.

![2018_12_13_12_17_34_export_wizard](https://user-images.githubusercontent.com/1867607/49935403-3a07a680-fed1-11e8-8cd2-528b9e7969f2.png)

@stempler @thorsten-reitz It might make sense to also add that option to the `Export project archive` dialog.

**Comment by thorsten-reitz** [comment]
@florianesser I agree ... and it should be checked by default IMHO :)

**Comment by Kate-Lyndegaard** [comment]
Yes, I am using shapefiles. Well, this explains my problem. I think it would be great to have the added option in the export dialog as well.
